README.md

```diff
@@ -6,7 +6,7 @@ This project allows you to unit test your Camunda Platform 8 BPMN processes. It
 and provide you with a set of assertions you can use to verify your process behaves as expected.
 
 > [!Note]
-> Heads up! We are building a new Java testing library for Camunda 8.6. The new library will replace Zeebe Process Test.
+> Heads up! We are building a new Java testing library for Camunda 8.6+. The new library will replace Zeebe Process Test eventually.
```

Comment:
@saig0 Will we release something usable already in 8.6 or was the entire topic postponed to 8.7?